code/modules/mob/living/carbon/human/life.dm

```diff
@@ -81,9 +81,11 @@
 
 		handle_medical_side_effects()
 
-		if(!client && !mind)
-			species.handle_npc(src)
-
+		// If we have an AI, and we're in a fugue state or without client.
+		// Give control to AI.
+		if(istype(ai) && ((stat & FUGUE) || (!client && !mind && species)))
```

Comment:
Noting that having the FUGUE check here would prevent an AI from toggling off and then back on again

Reply:
Not sure I understand why that matters since there's no toggle state machine on an AI.

Reply:
Yet. You are solidifying the inability to allow it in the future